packages/react/src/create-strict-api/__tests__/index.test.tsx

```diff
@@ -357,13 +365,19 @@ describe('createStrictAPI()', () => {
       });
 
       const { getByTestId } = render(
-        <Button
-          // @ts-expect-error -- Errors because `color` conflicts with the `XCSSProp` schema–`color` should be a css variable.
-          xcss={styles.primary}
```

Comment:
This no longer errors which was unexpected. I'm not sure why. But it is a type error in the `cssMap` call above, and it is also a type error inline (below). So I'm confident to let it slide.

Reply:
So luckily this doesn't allow a `styles.primary` with more keys on it than desired (eg. if `createStrictAPI<'color' | 'backgroundColor'>` and `XCSSProp<'color'>` are mismatched, but it doesn't deeply check the values.

If there was ever a need for say `<Button xcss>` to not allow a specific background/color via the `XCSSProp<…>` generic, it's probably all going to have to be rewritten—but maybe that's a good line in the sand to draw for design, we don't care about shipping design opinions, the component is the design opinion, we care about shipping boundaries to not cross so we can evolve the system over time, which may include things you don't like…

Reply:
There may be a need to restrict values in the future, potentially. But at the same time maybe it doesn't matter as, at least, createStrictAPI constrains the values. Also noting that the inline object was fine 🤔.

Reply:
Fuck yeah.